tests/unit/store/waveform.spec.js

```diff
@@ -191,42 +175,39 @@ describe("store/waveform", () => {
         context
       );
       await bound_ping_get_waveform_data();
-      expect(mocked_axios.history.get.length).toBe(1);
+      expect(mocked_axios.history.get).toHaveLength(1);
       expect(mocked_axios.history.get[0].url).toMatch(
         "http://localhost:4567/get_available_data"
       );
     });
-    test("When the start_get_waveform_pinging action is dispatched, Then setInterval is invoked and the interval ID stored in Vuex", async () => {
+    test("Given that the axios get method to respond with http status 200, When the start_get_waveform_pinging action is dispatched, Then setInterval is invoked and the interval ID stored in Vuex", async () => {
```

Comment:
it wasn't all axios `get` methods that were mocked---just `/get_available_data`. please update the test title